src/web/routes/application/check/post.js

```diff
@@ -54,6 +54,7 @@ const postCheck = (config) => async (req, res, next) => {
     const err = new Error('Error posting the request')
     err.statusCode = httpStatus.INTERNAL_SERVER_ERROR
     err.error = error.error
+    err.stack += '\nCaused by: ' + error.stack
```

Comment:
This makes sure that the stack trace of the original error is added to the stack of the new error and hence visible in the logs (not completely related to this PR but useful for diagnosing problems contacting the Claimant service).